std_msgs/Header header
float64 cam_dt_reader            # cam_dt_reader reading
float64 cam_dt_reader_var   # cam_dt_reader_var reading
